prov/verbs/src/verbs_domain.c

```diff
@@ -104,6 +109,12 @@ static int fi_ibv_domain_close(fid_t fid)
 
 	ofi_mr_cache_cleanup(&domain->cache);
 
+	if (domain->eq)
+	{
+		fi_ibv_eq_dettach_domain(domain->eq, domain);
+		domain->eq = NULL;
```

Comment:
maybe we should move setting / clearing domain->eq into the attach/detach functions.
nit: detach only has 1 t.